client/spec/monitoring_spec.js

```diff
@@ -471,4 +471,20 @@ describe('monitoring', function() {
         monitoring.actionOnDeskSingleView();
         expect(monitoring.getSingleViewItemCount()).toBe(8);
     });
+
+    it('can display respective single view title', function() {
```

Comment:
I think this can be merged with the above test `can display desk content in desk single view`.